webots_ros2_universal_robot/setup.py

```diff
@@ -31,7 +32,7 @@
 data_files.append(('share/' + package_name + '/worlds', worlds))
 data_files.append(('share/' + package_name + '/worlds/textures', textures))
 data_files.append(('share/' + package_name, ['package.xml']))
-
+data_files.append(('share/' + package_name + '/resource', ['resource/view_robot_dynamic.rviz']))
 
 setup(
     name=package_name,
```

Comment:
It would be nice to create a README.md file for this package (including the information from http://wiki.ros.org/webots_ros2/Tutorials/webots_ros2_universal_robot) to explain the difference between the two rviz examples.